tests/queries/0_stateless/02961_drop_tables.sql

```diff
@@ -0,0 +1,19 @@
+DROP DATABASE IF EXISTS 02961_db;
+CREATE DATABASE IF NOT EXISTS 02961_db;
+
+DROP TABLE IF EXISTS 02961_db.02961_tb1;
+CREATE TABLE 02961_db.02961_tb1 (id UInt32) Engine=Memory();
+
+DROP TABLE IF EXISTS 02961_db.02961_tb2;
+CREATE TABLE 02961_db.02961_tb2 (id UInt32) Engine=Memory();
+
+DROP TABLE IF EXISTS 02961_db.02961_tb3;
+CREATE TABLE 02961_db.02961_tb3 (id UInt32) Engine=Memory();
+
+SHOW TABLES FROM 02961_db;
+
+DROP TABLE IF EXISTS 02961_db.02961_tb1, 02961_db.02961_tb2, 02961_db.02961_tb3;
```

Comment:
Does it also support tables from different databases? If so, add some different databases for test.

Reply:
ok